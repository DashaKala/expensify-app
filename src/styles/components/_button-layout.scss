.button-layout {
  background: $blue-grey;
  border: none;
  color: $burlywood;
  display: inline-block;
  font-size: $l-font-size;
  font-weight: bolder;
  line-height: 1;
  padding: $s-size;
  text-decoration: none;
}

.button--link {
  background: none;
}

.button--secondary {
  background: $burlywood;
  color: $blue-grey;
}
