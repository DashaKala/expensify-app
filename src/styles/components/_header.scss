.header {
  background: $blue-grey;
}

.header__content {
  align-items: center;
  display: flex;
  justify-content: space-between;
  padding: $s-size 0;
}

.header__title {
  color: $burlywood;
  text-decoration: none;

  h1 {
    font-weight: bolder;
    margin: 0;
  }
}