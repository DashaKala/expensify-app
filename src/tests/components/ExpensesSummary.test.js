import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render ExpensesSummary for 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expenseTotal={235}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary for multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={5} expenseTotal={235125}/>);
    expect(wrapper).toMatchSnapshot();
});























