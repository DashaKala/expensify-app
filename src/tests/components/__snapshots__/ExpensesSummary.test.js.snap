// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary for 1 expense 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
        1
      </span>
       
      expense
       totalling 
      <span>
        $2.35
      </span>
    </h1>
    <div
      className="page-header__actions"
    >
      <[object Object]
        className="button-layout"
        to="/create"
      >
        Add Expense
      </[object Object]>
    </div>
  </div>
</div>
`;

exports[`should render ExpensesSummary for multiple expenses 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
        5
      </span>
       
      expenses
       totalling 
      <span>
        $2,351.25
      </span>
    </h1>
    <div
      className="page-header__actions"
    >
      <[object Object]
        className="button-layout"
        to="/create"
      >
        Add Expense
      </[object Object]>
    </div>
  </div>
</div>
`;
